Sources/Decoder.swift

```diff
@@ -157,6 +157,9 @@ public struct Decoder {
                 
                 for subJSON in jsonArray {
                     guard let model = T(json: subJSON) else {
+                        if T.self is ArrayPartiallyDecodable.Type {
```

Comment:
No Worries, I clearly understand you don't want to include this feature in the repo, it's being a large debate on this, honestly I just tried to find a generic way to solve a problem that came to me and that potentially could happen to other users while keeping it in the same Repo, I see it just as something useful you could use if needed, the solution **is non intrusive** and at the end of the day _(as everything in life)_ has requirements in order to work _(for me it makes sense)_

I will try to find some other way to solve the problem that I still have and I haven't been able to work around any other way.

Thanks for the time though, and no issues, everything's cool you have the right to push the project the best direction you think it should be. 👍